packages/blade/src/components/ActionList/ActionListItem.tsx

```diff
@@ -114,6 +114,31 @@ type ActionListSectionProps = {
   _sectionChildValues?: string[];
 } & TestID &
   DataAnalyticsAttribute;
+
+const _ActionListSectionTitle = ({
+  title,
+  isInsideVirtualizedList = false,
+}: {
+  title: string;
+  isInsideVirtualizedList?: boolean;
+}): React.ReactElement => {
+  return (
+    <StyledActionListSectionTitle
+      {...makeAccessible({
+        hidden: !isInsideVirtualizedList,
+        role: isInsideVirtualizedList ? 'heading' : undefined,
```

Comment:
Why role heading only when inside virtualized list? shouldnt a SectionTitle always be heading?

Reply:
in case of virtualized list we don't have wrapper which have role group. so have given it role of heading....

but in case of normal heading   we're announcing title as group label so we can hide this